core/src/main/java/io/questdb/std/BitSet.java

```diff
@@ -45,6 +45,13 @@ public int capacity() {
         return words.length * BITS_PER_WORD;
     }
 
+    public void checkCapacity(int wordsRequired) {
```

Comment:
This method expects the number of words (`long`), not the number of bits, so it's better to keep it private. Why do you need the method? Set methods grow the internal array when needed and get method tolerates out of range index as the input.